src/bp-messages/classes/class-bp-rest-group-messages-endpoint.php

```diff
@@ -158,6 +158,7 @@ public function create_item( $request ) {
 					$total_threads = BP_Messages_Thread::get(
 						array(
 							'include_threads' => array( $group_thread ),
+							'per_page'        => - 1
```

Comment:
@janakkaneriya007 Why we need to pass par_page `-1`. We already have total query separate without limit. 
Just pass `per_page=>1` and also pass `count_total` args

Reply:
@dipskakadiya We can use the newly added parameter (is_deleted) to optimize the query